quartz/plugins/transformers/ofm.ts

```diff
@@ -213,6 +217,35 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>>
         })
       }
 
+      if (opts.inlineFootnotes) {
+        // Replaces ^[inline] footnotes with regular footnotes [^1]:
+        const footnotes: Record<string, string> = {}
+
+        // Replace inline footnotes with references and collect definitions
+        const result = (src as string).replace(
+          inlineFootnoteRegex,
+          (_match: string, content: string) => {
+            const id = `inline-${Math.random().toString(36).substring(2, 8)}`
```

Comment:
Wouldn't this result in inconsistent links between deployment?

Reply:
Yeah, but I didn't think anyone would expect stable links to a footnote. 

Alternatively they could be assigned numbers counting up. Still sensitive to ordering but more stable.

Reply:
Changed now

Reply:
probably just have a small counter per file to be the most consistent with this.

Reply:
`Object.keys(footnotes).length` counts the amount of inline footnotes. How does the counter you have mind differ?

Reply:
oh i was thinking about the id, can just be `inline-0`, `inline-1`, etc.

Reply:
> oh i was thinking about the id, can just be `inline-0`, `inline-1`, etc.

But that's exactly what it does 😄. `footnotes[id] = content.trim()` increases the size of `footnotes` each iteration. The only difference is it starts at 1.

Reply:
but you are using a random string then slice it? thats diff from counter no? (I could be wrong here)

Reply:
That's old code